### ISSUE TITLE
reorganize NEWS

### ISSUE THREAD

**Issue by MichaelChirico** [r-lib/lintr#2223]
Implementation looks good.
Should be put in a more prominent position in NEWS imo.
Technically, it is a breaking change. Or we could add a Section "Experimental features"

_Originally posted by @AshesITR in https://github.com/r-lib/lintr/issues/2177#issuecomment-1744433777_
 
I also think we'd do well to have sections like "precision improvements" "recall improvements" that fit better with the nature of the package.

**Comment by AshesITR** [comment]
Apart from ML specialists, I'm afraid noone will understand precision / recall out of the box 😅

Would we consider false positives a bug?
If so, maybe improvements (more lints) vs. bug fixes (less lints)?

**Comment by MichaelChirico** [comment]
I think calling it a "bug" mixes up with what I consider bugs for lintr, namely that running the tool throws an unexpected error.

"Fixed false positives" feels a bit wordy too. precision/recall are the most concise descriptions... we could have a note at the top of the news explaining what they mean?

Otherwise we could lean into being very explicit "Code that wasn't meant to lint, now doesn't" "Code that was meant to lint, now does" along those lines

**Comment by MichaelChirico** [comment]
Using Type I/Type II error terminology might be understood by a wider audience of R users but I always have to look up which is which

**Comment by AshesITR** [comment]
I like "Fixed false positives" (and "Improved true positives"?)
